test/components/grade-result/d2l-grade-result-student-grade-preview.vdiff.js

```diff
@@ -176,7 +177,6 @@ describe('presentational-with-grade-preview', () => {
 			labelText: 'Overall Grade',
 			scoreNumerator: '5',
 			scoreDenominator: '20',
-			isGradeAutoCompleted: true,
```

Comment:
This [attribute was removed](https://github.com/BrightspaceGraveyard/grade-result/pull/85) so does nothing.